### PR TITLE
Issue 113 batch support for bigquery

### PR DESCRIPTION
**Pull request by YaroslavTir**
Fixes https://github.com/OHDSI/DatabaseConnector/issues/113

### CODE REVIEW

File: R/DatabaseConnector.R [hunk 1]
```diff
@@ -26,6 +26,21 @@ NULL
 
 .onLoad <- function(libname, pkgname) {
   rJava::.jpackage(pkgname, jars = "DatabaseConnector.jar", lib.loc = libname)
+  rJava::.jpackage(pkgname, jars = "api-common-1.8.1.jar", lib.loc = libname)
```
Comment (schuemie) [inline R/DatabaseConnector.R:29]: This is not how the .jpackage function works. Please read the rJava documentation

**Comment by schuemie** [comment]
I have serious concerns about adding this many external jar files. Please make a case-by-case argument for each jar file why it absolutely has to be included. For those that must be included, they should go into the DatabaseConnectorJars package.

Also, please target the develop branch.

File: R/DatabaseConnector.R (hunk 1 above)
Comment (YaroslavTir) [reply R/DatabaseConnector.R:29]: thank you point this 

> I have serious concerns about adding this many external jar files. Please make a case-by-case argument for each jar file why it absolutely has to be included. For those that must be included, they should go into the DatabaseConnectorJars package.
> 
> Also, please target the develop branch.

all this jar are necessary just to activate the stream  data loading functionality by big query: https://cloud.google.com/bigquery/streaming-data-into-bigquery
We are run a few analyses and found that it is not so easy to delete table afterword 
https://stackoverflow.com/a/43086239/1167673. So I am going to close this pull request and create another one.

**Comment by YaroslavTir** [comment]
> I have serious concerns about adding this many external jar files. Please make a case-by-case argument for each jar file why it absolutely has to be included. For those that must be included, they should go into the DatabaseConnectorJars package.
> 
> Also, please target the develop branch.

all this jar are necessary just to activate the stream  data loading functionality by big query: https://cloud.google.com/bigquery/streaming-data-into-bigquery
We are run a few analyses and found that it is not so easy to delete table afterword 
https://stackoverflow.com/a/43086239/1167673. So I am going to close this pull request and create another one.

**Pull request closed without merging**